frontend/lib/pages/loc-confirmation.tsx

```diff
@@ -12,13 +13,80 @@ import { EmailAttachmentForm } from '../email-attachment';
 import { EmailLetterMutation } from '../queries/EmailLetterMutation';
 import { BigList } from '../big-list';
 import { USPS_TRACKING_URL_PREFIX } from "../../../common-data/loc.json";
+import { SquareImage } from './data-driven-onboarding';
+
+const LetterViaEmailInstructions = `If you want to send your Letter of Complaint to your landlord and/or management company via email, download the PDF and include it as an attachment to your regular email.`
+
+const SanitationGuidelines = () => {
+
+  const [isExpanded, toggleExpansion] = useState(false);
+
+  return (
+    <div className="jf-sanitation-guidelines notification is-warning">
+      <div>
+        Please be aware that letting a repair-worker into your home to make repairs may expose you to the Covid-19 virus. 
+        In order to follow social distancing guidelines and to limit your exposure, please follow these steps to stay as safe as possible.
+        {' '}<button className={classnames("button","is-text","is-paddingless","is-uppercase", isExpanded && "is-hidden")} 
+              onClick={() => toggleExpansion(true)}>
+          Learn More
```

Comment:
Same accessibility comments regarding "Learn more" here.  Since we don't have an abstraction layer getting in the way here like we do with the DDO cards, it should be easy to turn this into something like `Learn more <span "jf-sr-only">about funky COVID-19 sanitization precautions</span>` so that the extra text is accessible to ATs.